src/main/java/com/amazon/opendistroforelasticsearch/security/tools/AuditConfigMigrater.java

```diff
@@ -48,7 +48,7 @@
 
     public static void main(String[] args) {
         options.addOption(Option.builder("s").argName("source").hasArg().desc("Path to elasticsearch.yml file to migrate. If not specified, will try to lookup env " + ES_PATH_CONF_ENV + " followed by lookup in current directory.").build());
-        options.addOption(Option.builder("oad").argName("output-audit-dir").hasArg().desc("Output directory to store the generated " + AUDIT_YML + " file. To be uploaded in the index, the file must be present in plugins/opendistro_security/securityconfig/ or use securityadmin tool.").build());
+        options.addOption(Option.builder("oad").argName("output-audit-dir").hasArg().desc("Output directory to store the generated " + AUDIT_YML + " file. To be uploaded in the index, the file must be present in plugins/opendistro-security/securityconfig/ or use securityadmin tool.").build());
```

Comment:
Can you also use the public constant `PLUGIN_NAME` defined in `src/main/java/com/amazon/opendistroforelasticsearch/security/OpenDistroSecurityPlugin.java`